canonical/juju-dashboard
# WD-8521 - Make secrets table responsive

## Done

- Make secrets table responsive

## QA

- Go to the secrets table and check that it looks reasonable at different screen sizes.

## Details

- https://warthogs.atlassian.net/browse/WD-8521

## Screenshots

<img width="312" alt="Screenshot 2024-02-19 at 2 44 38 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/9d057e35-66c9-4466-973c-edf1303e8a0b">
<img width="1002" alt="Screenshot 2024-02-19 at 2 46 26 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/47b26008-1b29-441c-937d-ec5d97bfc73e">
<img width="1665" alt="Screenshot 2024-02-19 at 2 46 48 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/1ebaab9a-18eb-43fd-b0ed-82617d8357d1">
